docs/blocks/ageverification.md

```diff
@@ -0,0 +1,33 @@
+## Age Verification
+The age verification block is automatically integrated into every page delivered if the cookie sazAgeOK is missing.<br>
```

Comment:
Can you try to explain it a bit more - just another sentence or two about how the cookie is working and what happens with accepting vs rejecting the question (redirect etc.).